src/index.js

```diff
@@ -0,0 +1,193 @@
+'use strict'
+
+const {
+  UnixFS: { unmarshal: decodeUnixFs }
+} = require('ipfs-unixfs')
+
+const { concurrency, codecs, blocksTable, primaryKeys, carsTable, publishingQueue } = require('./config')
+const { logger, elapsed } = require('./logging')
+const { openS3Stream } = require('./source')
+const { readDynamoItem, writeDynamoItem, deleteDynamoItem, publishToSQS, cidToKey } = require('./storage')
+const { forEach } = require('hwp')
+
+function decodeBlock(block) {
```

Comment:
Since we cut link indexing from this version, and link indexing will eventually be in another Lambda, let’s not decode the block data or deal with codecs at all.

Reply:
I've modified the function in order to disable codec disabling behing an environment variable.